src/omero_cli_render.py

```diff
@@ -807,10 +807,20 @@ def test_per_image(self, client, img, force, thumb):
         if error:
             error = str(error).split("\n")[0]
         elif thumb:
+            ctx = {'omero.group': str(img.details.group.id.val)}
             tb = client.sf.createThumbnailStore()
             try:
-                tb.setPixelsId(int(pixid), ctx)
-                tb.getThumbnailByLongestSide(rint(96), ctx)
+                has_rendering_settings = tb.setPixelsId(int(pixid), ctx)
+                if not has_rendering_settings:
+                    try:
+                        tb.resetDefaults(ctx)
+                        tb.setPixelsId(int(pixid), ctx)
+                        tb.getThumbnailByLongestSide(rint(96), ctx)
```

Comment:
Maybe the value of ``omero.client.browser.thumb_default_size`` should be used

Reply:
That makes sense and would be in agreement with the meaning of the configuration. Do you want this change to be carried out as part of this bug fix PR or should it be tackled separately?

Reply:
I don't think it needs to be done in this PR since the value ``96`` was already used.